docs/sources/viewing-json-logs/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/logs/viewing-json-logs/
+description: Learn how to view JSON formatted logs in Logs Drilldown.
+keywords:
+  - Logs
+  - JSON
+  - Formatting
+menuTitle: Viewing JSON logs
+title: Viewing JSON logs
+weight: 400
+---
+
+# Logs Drilldown JSON viewer
+You can easily view and interact with your JSON formatted logs  using the Logs Drilldown JSON viewer. This view will help you read your JSON style logs, and filter through them to make your related dashboards more relevant and focused.
+
+
+{{< admonition type="note" >}}
+Logs Drilldown JSON Viewer is an experimental feature. Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided. To use this feature, you must be running Loki 3.5.0 or later.
+{{< /admonition >}}
+
+## Viewing JSON logs
+
+To interact with the JSON view, select the **Show Logs** button for your service in Logs Drilldown. 
+
+{{< figure alt="JSON Table viewer with selector and include exclude highlighted" width="500px" align="center" src="/media/docs/explore-logs/show-logs.png" caption="Select the 'Show Logs' button on your service" >}}
+
+From there, select **JSON** in the format menu in the top right toolbar. This will show your logs in a structured, collapsible way, enabling you to sort, filter, and otherwise adjust your log data in the visualizations for your logs.
```

Comment:
```suggestion
From there, select **JSON** in the format menu for the Logs. This will show your logs in a structured, collapsible way, enabling you to sort, filter, and otherwise adjust your log data in the visualizations for your logs.
```